extensions/amp-story-interactive/0.1/amp-story-interactive-quiz.css

```diff
@@ -181,12 +184,18 @@
 .i-amphtml-story-interactive-quiz-option::before {
   content: '' !important;
   position: absolute !important;
+  left: 0px !important;
```

Comment:
@gmajoulet this should have `left: 0px` always since it's the hint element that needs to be aligned with the parent at all times